src/constants/v2.ts

```diff
@@ -5,6 +5,7 @@ export const FARM_CONTRACTS: { readonly [chainId in ChainId]?: Array<string> } =
   [ChainId.BSCTESTNET]: [],
   [ChainId.RINKEBY]: ['0x13c8F670d3bbd4456870a2C49Bb927F166A977Bd'],
   [ChainId.ROPSTEN]: [],
+  [ChainId.MATIC]: ['0x5C503D4b7DE0633f031229bbAA6A5e4A31cc35d8'],
```

Comment:
Move to networks const folder

Reply:
we have agent release for tomorrow, will refactor later